packages/ubuntu_wsl_setup/lib/app.dart

```diff
@@ -31,9 +32,20 @@ class UbuntuWslSetupApp extends StatelessWidget {
       debugShowCheckedModeBanner: false,
       localizationsDelegates: localizationsDelegates,
       supportedLocales: AppLocalizations.supportedLocales,
-      home: reconfigure
-          ? UbuntuWslReconfigureWizard(initialRoute: initialRoute)
-          : UbuntuWslSetupWizard(initialRoute: initialRoute),
+      home: buildWizard(context),
     );
   }
+
+  Widget buildWizard(BuildContext context) {
+    switch (variant) {
+      case Variant.WSL_SETUP:
+        return UbuntuWslSetupWizard(initialRoute: initialRoute);
+      case Variant.WSL_CONFIGURATION:
+        return UbuntuWslReconfigureWizard(initialRoute: initialRoute);
+      case null:
+        return const SizedBox.shrink();
```

Comment:
For the time being, the window is empty during the client variant query. If we want a "busy wait" screen, we could inject it here.

Here's a plain centered busy indicator with an artificial few seconds client variant query delay for testing.

https://user-images.githubusercontent.com/140617/140540281-fc5bc5bf-d6d3-48d1-a59d-8895a3abdc91.mp4

Reply:
looks great as a placeholder :) Indeed, we can hook that up later on